ovs-driver/src/main/java/org/opendaylight/unimgr/mef/nrp/ovs/activator/OvsActivatorHelper.java

```diff
@@ -163,4 +168,18 @@ private IngressBwpFlow getIngressBwpFlow() {
     protected boolean isIBwpConfigured() {
         return getIngressBwpFlow() != null;
     }
+
+    public static void validateExternalVLANs(List<EndPoint> endPoints) throws ResourceNotAvailableException {
+        final Set<Integer> vlans = new HashSet<>();
+        endPoints.stream().forEach(endPoint -> {
```

Comment:
if you use streams to do that, please use 
map method
ePs.stream.map{eP -> id).filter(id >=0).collect(toSet)

Reply:
will be rewritten